### ISSUE TITLE
Change use of promisify of `request` and `request`.defaults to work around a newrelic bug

### ISSUE THREAD

**Issue by jrgm** [mozilla-services/ip-reputation-js-client#7]
Hey @g-k,

You might not access the first two of these, but we ran into a bug with using newrelic + bluebird. 
- https://github.com/mozilla/fxa-customs-server-private/issues/43
- https://support.newrelic.com/tickets/297628
- https://github.com/jrgm/newrelic-bluebird-request-node8-bug

... but it would useful if you changed the reference to `request.defaults` like this:

- change https://github.com/mozilla-services/ip-reputation-js-client/blob/master/lib/client.js#L8 to do:
```
const _request = require('request'));
const request = Promise.promisify(_request);
```
and later call `.defaults` on the `_request` object at https://github.com/mozilla-services/ip-reputation-js-client/blob/master/lib/client.js#L43.

Ping me on irc or here if you have questions.

**Comment by jrgm** [comment]
I should have said more. This is not a production bug, but it does prevent moving to use newrelic in production for fxa-customs-server because of a bug with newrelic when combined with bluebird's promisify.

**Comment by g-k** [comment]
Thanks @jrgm! I'll do that first thing tomorrow.

**Comment by g-k** [comment]
OK should be fixed in >=2.2.1

@jrgm ping me if the updated version doesn't work.
